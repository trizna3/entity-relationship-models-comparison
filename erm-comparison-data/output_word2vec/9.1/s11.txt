airplane-rockets;airplanes
fight model-EMPTY
marine role-EMPTY
rocket type-EMPTY
transport model-EMPTY
marine-soldiers
airplane model-EMPTY
CHANGE_CARDINALITY:S-rockets-fighters
REBIND_1NN1_TO_MN:S-fighters
CHANGE_CARDINALITY:S-rockets-airplanes
CHANGE_CARDINALITY:S-airplanes-rockets
CONTRACT_11_ASSOCIATION:S-rockets-airplanes
