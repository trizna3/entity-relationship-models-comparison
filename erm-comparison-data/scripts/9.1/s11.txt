entity set airplanes {
	"name"
};

entity set "fighters" {
};

entity set "transport planes" {
	"person capacity"
};

entity set rockets {
	"name"
};

entity set material {
	"name"
};

entity set soldiers {
	"name"
	"actual health"
	"maximal health"
	"strength"
	"agility"
	"inteligence"
};

entity set "soldier types" {
	"name"
	"maximal health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

association between 
{
	airplanes ONE
	fighters MANY
}{};

association between 
{
	airplanes ONE
	"transport planes" MANY
}{};

association "load" between 
{
	"transport planes" MANY
	material MANY
}{
	"volume"
	"weight"
};

association "fighter rockets" between 
{
	fighters MANY
	rockets MANY
}{
	"count"
};

association between 
{
	airplanes MANY
	soldiers MANY
}{};

association between 
{
	soldiers MANY
	"soldier types" ONE
}{};
