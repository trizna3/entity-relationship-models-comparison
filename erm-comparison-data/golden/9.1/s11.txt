airplane model-
fight model-fighters
rocket type-rockets
transport model-transport planes
airplane-airplanes
marine-soldiers
marine role-soldier types
CHANGE_CARDINALITY:S-transport planes-material
CHANGE_CARDINALITY:S-material-transport planes
CONTRACT_11_ASSOCIATION:S-transport planes-material